pfurl/pfurl.py

```diff
@@ -1112,9 +1112,13 @@ def pushPath_compress(self, d_msg, **kwargs):
             d_ret['status'] = d_ret['remoteServer']['status']
             d_ret['msg']    = d_ret['remoteServer']['msg']
         else:
-            d_ret['status'] = d_ret['remoteServer']['decode']['status']
-            d_ret['msg']    = d_ret['remoteServer']['decode']['msg']
-            # raise Exception('Invalid Response')
+            try:
+                d_ret['status'] = d_ret['remoteServer']['decode']['status']
+                d_ret['msg'] = d_ret['remoteServer']['decode']['msg']
+            except:
```

Comment:
I was expecting something different.  Are we expecting something to be a dictionary here and it isn't?  Can we tell that more elegantly that catching a generic exception?  Can we also return the error message that's returned from the server rather than saying a non-json response?